src/app/page/UserProfile/UserOverview.jsx

```diff
@@ -59,9 +59,11 @@ const UserStatus = ({ profile, setProfile, view }) => {
     }
   };
 
-  if (loaded && !error) {
-    handleScriptLoad();
-  }
+  useEffect(() => {
+    if (loaded && !error) {
+      handleScriptLoad();
+    }
+  });
```

Comment:
useEffect should include at least an empty array as its second argument. If this effect is dependent on loaded or error changing, then those variables should be included in that dependency array

Reply:
Got it. Addressed in 78c8ef5. I don't think the effect should be dependent on loaded or error as the fallback mechanism is to just allow freeform entry of address/location.

Reply:
If `handleScriptLoad` is NOT dependent on `loaded` and `not error`, then there should be no reason to use these variables in an `if` block

Reply:
Sorry, I misread the question. handleScriptLoad is dependent on `loaded` and `not error`.